src/network/os1/client/codec/game/InterfaceClickDecoder.ts

```diff
@@ -3,16 +3,21 @@ import type Packet from '#/io/Packet.ts';
 import type ClientMessage from '#/network/client/ClientMessage.ts';
 import GameClientLimit from '#/network/client/codec/game/GameClientLimit.ts';
 import MessageDecoder from '#/network/client/codec/MessageDecoder.ts';
+import NoOp from '#/network/client/model/game/NoOp.ts';
 import Logout from '#/network/server/model/game/Logout.ts';
 
-export default class LogoutDecoder extends MessageDecoder {
+export default class InterfaceClickDecoder extends MessageDecoder {
```

Comment:
InterfaceClick is called IF_BUTTON on their end, for InterfaceButton - we can do IfButton or InterfaceButton with a comment